src/index.scss

```diff
@@ -1,30 +1,61 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900;1000&display=swap');
 
 $blue: #465491;
+$white: #fff;
```

Comment:
We've decided to use just 'white' which browsers support instead of a variable for white

Reply:
Okay! Could this be reflected in this PR?